main.py

```diff
@@ -60,8 +57,14 @@ def get_edit(request: Request, item_id: int, db: Session = Depends(get_db)):
 
 
```

Comment:
Consider adding validation for the 'notes' field, such as max length or content type checking.